### PR TITLE
Add loading button

### PR DESCRIPTION
**Pull request by chinmaikaveti2001**
Added special and an animation loading button

 Adds special ,hover and an animation loading button using css  for open source  sbuttons .....

Issue: #436

### CODE REVIEW

**Comment by HADES-01** [comment]
@chinmaikaveti2001 You have to add the buttons to the less files and not he css files because they are generated from the less files. And also add them to the buttons-examples.js so that they can be rendered.

File: src/components/_special.less [hunk 1]
```diff
@@ -1,2 +1,28 @@
...
+
+.buttonload:hover {
+  background: #30e3cb;
+  background-image: -webkit-linear-gradient(top, #30e3cb, #2bc4ad);
+  background-image: -moz-linear-gradient(top, #30e3cb, #2bc4ad);
+  background-image: -ms-linear-gradient(top, #30e3cb, #2bc4ad);
+  background-image: -o-linear-gradient(top, #30e3cb, #2bc4ad);
+  background-image: linear-gradient(to bottom, #30e3cb, #2bc4ad);
+  text-decoration: none;
+}
```
Comment (HADES-01) [inline src/components/_special.less:28]: You should create a new file _loading.less in special folder and import it in here like the _play-game.less or _retro.less

File: index.html [hunk 2]
```diff
@@ -94,22 +94,23 @@
...
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-circle-o-notch fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-refresh fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-cog fa-spin fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+        <a class="buttonload" href="#"><i class="fa fa-spinner fa-pulse fa-1x fa-fw"></i>
+          <span class="sr-only">Loading...</span> LOADING</a>
+      </div>
```
Comment (HADES-01) [inline index.html:108]: You don't need to add the html here just take look at the [buttons-examples.js](https://github.com/shahednasser/sbuttons/blob/master/assets/js/buttons-examples.js) and add it to the buttons object with required propeties.

**Review by HADES-01 — CHANGES_REQUESTED** [review]

File: src/components/_special.less (hunk 1 above)
Comment (chinmaikaveti2001) [reply src/components/_special.less:28]: kk  @HADES-01

**Comment by HADES-01** [comment]
@chinmaikaveti2001  Please read [Contributing guide](https://github.com/shahednasser/sbuttons/blob/master/CONTRIBUTING.md) ->Instructions -> Point 2 for correct way to add new ideas

**Comment by shahednasser** [comment]
I'm closing this PR as it is somewhat messy. @chinmaikaveti2001 please read the contributing guidelines and try again.

**Comment by shahednasser** [comment]
@HADES-01 thank you for reviewing and helping the contributor!

@all-contributors please add @HADES-01 for review

**Pull request closed without merging**